src/main/java/mods/railcraft/common/gui/GuiHandler.java

```diff
@@ -42,7 +42,7 @@ public static void openGui(EnumGui gui, EntityPlayer player, World world, int x,
     public static void openGui(EnumGui gui, EntityPlayer player, World world, Entity entity) {
         if (Game.isHost(world)) {
             if (gui.hasContainer()) {
-                player.openGui(Railcraft.getMod(), gui.ordinal(), world, entity.getEntityId(), -1, 0);
+                player.openGui(Railcraft.getMod(), gui.ordinal(), world, entity.getEntityId(), Integer.MIN_VALUE, 0);
```

Comment:
This probably works, but if it doesn't we could always pack it into to gui.ordinal() parameter. That's how Forestry sends page numbers for its GUIs.

Reply:
Uhh, using an `~` bitwise operation may be too confusing for coders compared to `-`, though `-` is an incorrect operation.